### PR TITLE
feat: Line of route tracks

### PR DESCRIPTION
**Pull request by alainajaffer**
Set up the site to connect to the local rds url
Go to the map some services may not have track data so will show the old way I found that welsh services had the track data so followed the road

### CODE REVIEW

File: packages/api/client.ts [hunk 1]
```diff
@@ -367,6 +371,16 @@ export const getServiceStops = async (dbClient: Kysely<Database>, input: Service
         return [];
     }
 
+    const tracks = await dbClient
```
Comment (ljones92) [inline packages/api/client.ts]: Could we do this before the datasource query?

Comment (alainajaffer) [reply packages/api/client.ts]: done

File: packages/api/client.ts [hunk 2]
```diff
@@ -352,11 +352,25 @@ export type ServiceStopsQueryInput = {
...
+): Promise<ServiceStops | ServiceTracks> => {
     logger.info("Starting getServiceStops...");
 
+    const tracks = await dbClient
+        .selectFrom("tracks")
+        .select(["operatorServiceId as serviceId", "longitude", "latitude"])
+        .where("tracks.operatorServiceId", "=", input.serviceId)
+        .execute();
+
+    if (tracks && tracks.length > 0 && input.useTracks) {
```
Comment (ljones92) [inline packages/api/client.ts]: We could wrap the previous query in a check for `input.useTracks`, that way it won't run the query if not needed

File: packages/api/get-service-routes.ts [hunk 3]
```diff
@@ -89,12 +91,16 @@ const filterStops = (flattenedStops: ServiceStop[], direction: string) => {
 };
 
 export const formatStopsRoutes = async (
```
Comment (ljones92) [inline packages/api/get-service-routes.ts:96]: We could use a type predicate here:

```ts
const isServiceStops = (stops: ServiceStops | ServiceTracks): stops is ServiceStops =>
    !!(stops as ServiceStops)[0]?.dataSource;

export const formatStopsRoutes = async (
    stops: ServiceStops | ServiceTracks,
    // eslint-disable-next-line @typescript-eslint/require-await
): Promise<{ outbound: ServiceStop[] | ServiceTracks; inbound: ServiceStop[] }> => {
    if (isServiceStops(stops)) {
        const flattenedStops = flattenStops(stops);

        const outbound = filterStops(flattenedStops, "outbound");
        const inbound = filterStops(flattenedStops, "inbound");
        return { outbound, inbound };
    } else {
        return { outbound: stops, inbound: [] };
    }
};
```

Comment (alainajaffer) [reply packages/api/get-service-routes.ts:96]: done

File: packages/api/client.ts (hunk 2 above)
Comment (alainajaffer) [reply packages/api/client.ts]: done

**Review by ljones92 — APPROVED** [review]

**Pull request merged**